app/services/solidus_pay_tomorrow/client/base_service.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'httparty'
+
+module SolidusPayTomorrow
+  module Client
+    class BaseService
+      def self.call(*args, **kwargs, &block)
+        new(*args, **kwargs, &block).call
+      end
+
+      private
+
+      # Staging: 'https://api-staging.paytomorrow.com'
+      # Production: 'https://api.paytomorrow.com'
+      def api_base_url
+        ENV['API_BASE_URI']
```

Comment:
If the URLs are just 2 it's better not pass them from ENV and instead infer based on the environment.
E.g. if payment method environment is 'test' then use staing url, else use production.
Like this you avoid issues where a user might accidentally input the wrong URL (typical case: should the URL end in '/' or not?)

Reply:
Fixed in https://github.com/nebulab/solidus_pay_tomorrow/pull/3/commits/cc0f0c29f4c9f55c572361b14442ded375957302